### PR TITLE
Adds automation test app and instructions

### PR DESCRIPTION
**Pull request by mitchhentges**
* [x] I've added credential information to the releng private repo
* [x] Automation Test app created on Google Play
* [x] Automation Test app created on Amazon
* [x] `build.sh` script created to make generating new "Automation Test" apps a breeze (or, at least, as much of a breeze as possible)

### CODE REVIEW

File: app/.gitignore [hunk 1]
```diff
@@ -0,0 +1,14 @@
+*.iml
```
Comment (mitchhentges) [inline app/.gitignore]: This was generated by Android Studio

File: app/gradle.properties [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Project-wide Gradle settings.
```
Comment (mitchhentges) [inline app/gradle.properties]: This was generated by Android Studio

File: app/gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#Tue Oct 29 14:05:14 PDT 2019
```
Comment (mitchhentges) [inline app/gradle/wrapper/gradle-wrapper.properties]: This was generated by Android Studio

File: app/gradlew [hunk 4]
```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```
Comment (mitchhentges) [inline app/gradlew]: This was generated by Android Studio

File: app/gradlew.bat [hunk 5]
```diff
@@ -0,0 +1,84 @@
+@if "%DEBUG%" == "" @echo off
```
Comment (mitchhentges) [inline app/gradlew.bat]: This was generated by Android Studio

File: README.md [hunk 6]
```diff
@@ -46,18 +47,81 @@ source venv/bin/activate
...
+### Targeting Google Play
 
-1. Generate a Google Play Store p12 certificate. This certificate needs to have write access to the app you want to publish. In this context, "app" means Fennec, Fennec Beta or Fennec Nightly.
-1. Execute the steps defined in the section above.
-1. Download the latest signed builds. For instance, for Fennec Nightly: `./mozapkpublisher/get_apk.py --latest-nightly`
-1. 
-```sh
-./mozapkpublisher/push_apk.py --no-gp-string-update --track beta --credentials /path/to/your/googleplay/creds.p12 --service-account your-service-account@iam.gserviceaccount.com x86.apk arm.apk
+1. You'll need `mozapkpublisher` to authorize itself as a service account. To get credentials for the service account:
+    1. Go to the [Google Cloud Platform Console](https://console.cloud.google.com/)
```
Comment (JohanLorenzo) [inline README.md]: TIL we can generate these keys there! I thought it was just on the Google Play web app.

File: app/build.sh [hunk 7]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+if [ "$#" -ne 2 ]; then
+    echo "Usage: ./build.sh VERSION KEYSTORE_LOCATION"
+    echo "Example: ./build.sh 1 ~/keystore.jks"
+    exit 1
+fi
+
```
Comment (JohanLorenzo) [inline app/build.sh]: Nit: Assign `$1` and `$2` to named local variables so it's easier to follow what they do.

File: app/src/main/AndroidManifest.xml [hunk 8]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          package="org.mozilla.automationtest">
```
Comment (JohanLorenzo) [inline app/src/main/AndroidManifest.xml]: 👍

**Comment by mitchhentges** [comment]
Thanks @JohanLorenzo! I've defined names for the variables in `build.sh`, feel free to land when you're able

**Comment by JohanLorenzo** [comment]
Assigning this PR to myself so I don't lose track of it. @mitchhentges told me this PR is ready. I'll check if it neeeds to be rebased somehow. 

Thanks for bringing this up, @mitchhentges!

**Pull request closed without merging**